packages/network-controller/CHANGELOG.md

```diff
@@ -5,9 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [15.2.1]
 ### Changed
 - Bump dependency `@metamask/eth-query` from ^3.0.1 to ^4.0.0 ([#2028](https://github.com/MetaMask/core/pull/2028))
 
+### Fixed
+- Align `Provider` type to be compatible with `@metamask/eth-query` ([#2028](https://github.com/MetaMask/core/pull/2028))
```

Comment:
Okay sorry to go back on this 😅  I now think this should be a breaking change and we should bump to 16.0.0:

``` suggestion
## [16.0.0]
### Changed
- **BREAKING:** Bump dependency `@metamask/eth-query` from ^3.0.1 to ^4.0.0 ([#2028](https://github.com/MetaMask/core/pull/2028))
  - This is breaking because it changes the type of the EthQuery instance this controller creates internally and exports under the `getEthQuery` action. Please consult the [changelog for `@metamask/eth-query` 4.0.0](https://github.com/MetaMask/eth-query/blob/main/CHANGELOG.md#400) for more.
```

Reply:
No I think that makes sense! Updating now...

Reply:
Good call on making this breaking